src/command/http-command.ts

```diff
@@ -411,7 +411,7 @@ export class HttpCommand implements CommandInterface<HttpOptions> {
 				issuer: { ...cert.issuer },
 				subject: {
 					...cert.subject,
-					alt: cert.subjectaltname,
+					alt: cert.subjectaltname || null,
```

Comment:
Same here.

Reply:
My assumption comes from the `Cert` type, no real case as well.